exercises/ex1.cpp

```diff
@@ -5,44 +5,45 @@ using namespace std;
 
 int main()
 {
-  int week;
-
-  /* Input week number from user */
-  cout << "Enter week number(1-7): " << endl;
-  cin >> week;
-
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+	int week;
+
+	/* Input week number from user */
+  	cout << "Enter week number(1-7): " << endl;
+  	cin >> week;
+ 	switch (week)
+  	{
+		case (1):
+			cout << "Monday" << endl;
+		break;
+
+		case (2):
+			cout << "Tuesday" << endl;
+		break;
+
+		case (3):
```

Comment:
non sono necessarie le parentesi

(comunque questo esercizio è del vecchio laboratorio, il nuovo laboratorio prevedeva di implementare gli esercizi in python sotto la repository qd-starter-pack)

Reply:
Si lo so, ho fatto velocemente un paio di esercizi giusto per utillizzare git e github e vedere se riuscivo a fare tutto tramite git bash avendo collegato l'account github

Reply:
bravissimo